cmd/snap/cmd_quota_test.go

```diff
@@ -62,13 +62,19 @@ func makeFakeQuotaPostHandler(c *check.C, body, groupName, parentName string, sn
 		buf, err := ioutil.ReadAll(r.Body)
 		c.Assert(err, check.IsNil)
 
-		var snapNames []string
-		for _, sn := range snaps {
-			snapNames = append(snapNames, fmt.Sprintf("%q", sn))
+		switch action {
+		case "remove":
+			c.Check(string(buf), check.DeepEquals, fmt.Sprintf("{\"action\":\"remove\",\"group-name\":%q}\n", groupName))
```

Comment:
Will this work?
```suggestion
			c.Check(string(buf), check.DeepEquals, fmt.Sprintf(`{"action":"remove","group-name":%q}`, groupName))
```

Also check.Equals should work here and in the other place.